tests/integration-tests/tests/schedulers/test_slurm/test_slurm/mpi_job.sh

```diff
@@ -7,4 +7,4 @@
 #SBATCH --output=runscript.out
 
 module load intelmpi
-mpirun -n 6 IMB-MPI1 Alltoall -npmin 2
+mpirun -n 6 bash -c 'sleep 300' -npmin 2
```

Comment:
If we do this we are not using MPI at all because each node will launch a process that will sleep for a fixed amount of time, but no interaction will happen between the nodes.

Let's sync up about this.

Reply:
The test involves checking if cancellation of an MPI process does not leave any stray processes. The message exchange may not be needed but as discussed, this test "implicitly" covers IntelMPI tests so this PR has been opened pending follow up on reviewing the timing issue on Ubuntu2204: https://github.com/aws/aws-parallelcluster/pull/5666